### PR TITLE
More info for zypper install phase

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,12 +19,12 @@ RUN zypper addrepo http://download.opensuse.org/repositories/home:illuusio/openS
...
   zypper removerepo 'NON-OSS' && \
   # Package dependencies
-  time zypper --no-gpg-checks --non-interactive \
+  time zypper -vv --color --no-gpg-checks --non-interactive \
       # science contains latest Julia
       --plus-repo http://download.opensuse.org/repositories/science/openSUSE_Tumbleweed/ \
       # luarocks
       --plus-repo http://download.opensuse.org/repositories/devel:languages:lua/openSUSE_Tumbleweed/ \
-      install \
+      install --details --download-in-advance \
```
Comment (Mixih) [inline Dockerfile]: perhaps you should explain the `--download-in-advance` in the commit?

**Comment by jayvdb** [comment]
I've played around with all combinations of `-v` , `-vv` and `--details`.  Either `-v` or `--details` adds the list of all packages to be installed with precise information about where the (to be) *installed* package comes from, as data like:

(1)
```
R-KernSmooth                  
  2.23.15-11.1                  
  x86_64
  OSS                                                                                  
  openSUSE    
```

Either option also includes an entry for each *selected* package, indicating roughly the same thing in the form of:

(2)
```
Selecting 'libxml2-tools-2.9.4-4.1.x86_64' from repository 'OSS' for installation.
...
Selecting 'julia-0.6.0-56.8.x86_64' from repository 'http://download.opensuse.org/repositories/science/openSUSE_Tumbleweed/' for installation.
```

Using `-vv` seems to only add HTTP fetch information everywhere, which effectively provides similar information for each (to be) *installed* package in the form of:

(3)
```
Retrieving package perl-TimeDate-1.20-26.5.noarch (135/334),  45.5 KiB ( 93.1 KiB unpacked)
Retrieving: http://download.opensuse.org/tumbleweed/repo/oss/suse/noarch/perl-TimeDate-1.20-26.5.noarch.rpm [.done (6.9 KiB/s)]
```

Typically problems are caused by (to be) *installed* packages, which are dependencies of the *selected* packages.

Form (1) is the most sparse and resulting in the log always exceeding the size Travis naturally shows.

The most informative and *dense* information is form (3), but we cant have form (3) without also getting form (1). :(

As a result, I think we should have an ENV variable which can be enabled to add all of these flags, and they are disabled by default so that the logs normally fit within the Travis viewing size.

**Comment by jayvdb** [comment]
Here is what an intentionally broken install looks like , with all the lovely XML emitted into the log: https://travis-ci.org/jayvdb/docker-coala-base/builds/260167281

**Comment by yukiisbored** [comment]
ack https://github.com/coala/docker-coala-base/pull/149/commits/18a46703bec157bf340c7cd1892ff1437b25467e

**Comment by yukiisbored** [comment]
@rultor merge pls

**Comment by yukiisbored** [comment]
@rultor merge --do-not-fail

**Pull request merged**